proposals/0013-expanded-comparison-sampling.md

```diff
@@ -0,0 +1,432 @@
+* Status: **Under Consideration**
+* Planned Version: Shader Model 6.8, validator version 1.8
+* Issues: [30](https://github.com/microsoft/hlsl-specs/issues/30),
+          [81](https://github.com/microsoft/hlsl-specs/issues/81)
+
+## Introduction
+
+Sampler operations in HLSL perform either a comparison sampler or non-comparison
+ sampler operation.
+Comparison sample operations will perform the compare operation specified in the
+ sampler between the sampled value and a provided value and produce a 0 or 1
+ value indicating the failure or success of the comparison respectively.
+In linear filtered samples, multiple values will be blended to produce the
+ complete sample operation result.
+
+HLSL has sampling operations which sample from a texture according to the
+ settings in the provided sampler object.
+The most useful forms of sampling involve the calculation of a MIP level
+ to determine which level(s) of detail(LOD) of the sampled texture should be
+ used and how they should be weighted.
+
+Different non-comparison sampler operations calculate the MIP level in different
+ ways:
+
+* `Sample(SamplerState S, float<num> Location)`
+  samples using LOD values that are determined implicitly using gradients
+  (sometimes called derivatives) representing how much the _Location_ changes
+  across neighboring pixels.
+* `SampleBias(SamplerState S, float<num> Location, float Bias)`
+   samples using the same implicitly-determined LOD with an
+   explicitly-provided _Bias_ value added to that result
+* `SampleGrad(SamplerState S, float<num> Location, float DDX, float DDY)`
+   samples using the LOD calculated using the explicitly-provided
+   _DDX_ and _DDY_ gradients.
+* `SampleLevel(SamplerState S, float<num> Location, float LOD)`
+   samples using an explicitly-provided _LOD_.
+
+Comparison sampler operations have fewer options for how the LOD is calculated:
+
+* `SampleCmp(SamplerComparisonState S, float<num> Location)`
+  performs a comparison sample using implicitly-determined LOD values.
+* `SampleCmpLevelZero(SamplerComparisonState S, float<num> Location, float cmpVal)`
+  performs a comparison sample using LOD 0
+* `SampleCmpLevel(SamplerComparisonState S, float<num> Location, float cmpVal, float LOD)`
+  performs a comparison sample using an explicitly provided _LOD_
+  (as of shader model 6.7)
+
+Additionally, the level of detail value can be calculated and retrieved directly
+ for non-comparison samplers:
+
+* `CalculateLevelOfDetailUnclamped(SamplerState S, float<num> Location)`
+  returns the raw LOD value determined implicitly using gradients representing
+  how much the _Location_ changes across neighboring pixels regardless of
+  whether it falls outside the valid range.
+* `CalculateLevelOfDetail(SamplerState S, float<num> Location)`
+  returns the implicitly-determined LOD value clamped to the valid range.
+
+There are no `CalculateLevelOfDetail` variants for comparison samplers.
```

Comment:
While all this detail is good, it doesn't seem to follow the advice in the newly introduced template for "Introduction".  Perhaps these details belong under "Motivation" instead?  Then "Introduction" can better follow the template's guidance: "10,000 ft view of the change being proposed. Try to keep to one paragraph and less than 10 sentences."

Reply:
will do